src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -37,10 +37,28 @@ function renderPaginationLink(data, index){
 
 	const episodeId = UTILS.convertToSafeString(title, "-");
 
-	const isActive = index === activeIndex;
+	const isActive 	= index === activeIndex;
+
+	let opacity = 1;
+	if(!isActive){
+		const decayDistance = 4;
+		const decayRate     = 1/decayDistance;
+		const fadeOffset    = Math.abs(index - activeIndex);
+		
+		opacity -= fadeOffset * decayRate;
```

Comment:
```suggestion
		opacity = 1 - (fadeOffset * decayRate);
```
as sexy as `-=` is, it implies that this function is going to be iterated over in a local loop or something, so I think keeping it pure assignment might be better.